lib/helpers.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * Extracts the measurement prefix from the configuration.
+ */
+const measurementPrefix = (config) => {
```

Comment:
This function is called for every event; and yet as far as I can see the result will always be the same. Would it be more efficient to calculate the prefix when `GoodInflux` is initialized? Then pass that string to `LineProtocol.format` for each event?

The same optimization could probably be made for metadata, I presume?

Reply:
Good point, I was debating this myself. It would definitely be more efficient. That said, it seems to perform fine from what I'm seeing in Grafana, and I like the nice, clean (event, config) signature of the formatters, ie. "here are the objects to format, give me back a formatted string" - exactly what a formatter should do, nothing more and nothing less. Moving formatting logic outside of `format()` sort of breaks single responsibility.

If LineProtocol were something more like a class, it could accept config as a constructor parameter and pre-compute those values?

Definitely some things to talk about - open to revisiting in a subsequent PR? Would like to get this in so that teams at CK can leverage Good Influx :)